certora/specs/erc4626.spec

```diff
@@ -0,0 +1,790 @@
+import "erc20.spec"
+import "StaticATokenLM.spec"
+
+using DummyERC20_aTokenUnderlying as underlying
+using StaticATokenLMHarness as SAT
+using AToken as ATok
+using SymbolicLendingPoolL1 as pool
+
+methods{
+    convertToAssets(uint256) returns (uint256) envfree
+    previewDeposit(uint256) returns(uint256) envfree
+    deposit(uint256, address) returns (uint256)
+    underlying.balanceOf(address) returns(uint256) envfree
+    ATok.balanceOf(address) returns(uint256) envfree
+    getULBalanceOf(address) returns(uint256) envfree
+    getATokenBalanceOf(address) returns (uint256) envfree
+    totalSupply() returns (uint256) envfree
+    assetsTotal(address) returns (uint256) envfree
+    totalAssets() returns (uint256) envfree
+    pool.getReserveNormalizedIncome(address) returns (uint256)
+    getStaticATokenUnderlying() returns (address) envfree
+    upperBound(uint256) returns (uint256) envfree
+    GetRAY() returns (uint256) envfree
+    ATok._underlyingAsset() returns (address) envfree
+    previewWithdraw(uint256) returns (uint256) envfree
+    previewRedeem(uint256) returns (uint256) envfree
+    maxRedeem(address) returns (uint256) envfree
+    maxDeposit(address) returns (uint256) envfree
+    ATok.allowance(address, address) returns (uint256) envfree
+    previewMint(uint256) returns (uint256) envfree
+    previewWithdraw(address) returns (uint256) envfree
+    maxWithdraw(address) returns (uint256) envfree
+}
+
+
+definition RAY() returns uint256 = 10^27;
+definition HALF_RAY() returns uint256 = 5 * 10^27;
```

Comment:
5 * 10^26

Reply:
this has been removed as it wasn't being used anyway.